com.ibm.jbatch.tck.dist.exec/tck.execution.for.ri.properties

```diff
@@ -52,17 +46,13 @@ jvm.options=-Dcom.ibm.jbatch.spi.ServiceRegistry.BATCH_THREADPOOL_SERVICE=com.ib
 #com.ibm.jbatch.spi.ServiceRegistry.DELEGATING_JOBXML_LOADER_SERVICE=com.ibm.jbatch.container.services.impl.DelegatingJobXMLLoaderServiceImpl
 #com.ibm.jbatch.spi.ServiceRegistry.CONTAINER_ARTIFACT_FACTORY_SERVICE=com.ibm.jbatch.container.services.impl.WeldSEBatchArtifactFactoryImpl
 #com.ibm.jbatch.spi.ServiceRegistry.DELEGATING_ARTIFACT_FACTORY_SERVICE=com.ibm.jbatch.container.services.impl.DelegatingBatchArtifactFactoryImpl
-
 #####
 # These properties configure the SigTest portion of the TCK
 #####
-
 # Point to SigTest (reminder IBM does not want to ship SigTest in any TCK distribution of our own packaging, but we can use it internally)
 sigtestdev.classes=../runSigTest/sigtestdev.jar
-
 # The Batch API being tested
 batch.api.classes=../runAgainstRI/jakarta.batch-api.jar
-
 # shipped by TCK
-javax.inject.classes=lib/jakarta.inject-api-1.0.jar
-javax.enterprise.util.classes=lib/jakarta.enterprise.cdi-api-2.0.1.jar
+jakarta.inject.classes=lib/jakarta.inject-api-1.0.jar
+jakarta.enterprise.util.classes=lib/jakarta.enterprise.cdi-api-2.0.1.jar
```

Comment:
These are fine for this PR.   Just observing they'll have to change before we can actually run anything, since we need these dependencies at a level with the jakarta.* packages.   I know these originally come from one of the POMs too.

Reply:
Yes, thanks for mentioning it. They should be 2.0.x and 3.0.x, shouldn't?